src/backend/src/form-responses/form-responses.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { FormResponse } from 'src/entities/formResponse.entity';
+import { FormResponsesService } from './form-responses.service';
+import { CreateFormResponseDto, CreateMultiFormResponsesDto } from './dto';
+
+@Controller('form-responses')
+export class FormResponsesController {
+  constructor(private readonly formResponsesService: FormResponsesService) {}
+
+  @Post()
+  async create(
+    @Body('userId') userId: number,
+    @Body('questionId') questionId: number,
+    @Body('formResponse') formResponseData: CreateFormResponseDto,
+  ): Promise<FormResponse> {
+    return this.formResponsesService.create(
+      userId,
+      questionId,
+      formResponseData,
+    );
+  }
+
+  @Post('all')
+  async createAll(
+    @Body('userId') userId: number,
+    @Body('formResponsesData') formResponsesData: CreateMultiFormResponsesDto,
```

Comment:
oh i see, you're using the create mutliform response data as an import type? I was under the impression that dtos should be the output type not the input type. But I could be wrong? We can do some research

Reply:
I think it can be used for both? A DTO is an object that defines how the data will be sent over the network.